README.md

```diff
@@ -0,0 +1,39 @@
+# WooCommerce Stripe Payment Gateway
+
+This is the official feature plugin for accepting payments via the Stripe payment gateway on a WooCommerce store.
+
+## Dependencies
+
+-   WooCommerce
+
+## Version support policy
+
+We adopt the L-2 version support policy for WordPress core strictly, and a loose L-2 policy for WooCommerce. See [more details](./docs/version-support-policy.md).
+
+## Development
+
+### Install dependencies & build
+
+-   `npm install`
+-   `composer install`
+-   `npm run build:webpack`, or if you're developing the client you can have it auto-update when changes are made: `npm start`
+
+If you run into errors with `npm install` it may be due to node version, try `nvm install` followed by `nvm use` then try again.
+
+When running the `composer install/update`, composer may prompt you for a GitHub OAuth token before it can fetch the `subscriptions` and `pre-orders` extensions from GitHub.
```

Comment:
Should we capitalize "Composer may..." for consistency in referring to tools?

Reply:
Done in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4031/commits/3985a5ead0bdbad13435fb4717ea76cc561d778d